eslint.js

```diff
@@ -103,7 +103,7 @@ module.exports = {
       after: true
     }],
     'space-before-blocks': 'error',
-    'space-before-function-paren': ['error', 'never'],
+    'space-before-function-paren': ['error', {'anonymous': 'never', 'named': 'never', 'asyncArrow': 'always'}],
```

Comment:
suggest to split the object into separate lines and remove quote on key.

example:
https://github.com/hexojs/eslint-config-hexo/blob/8ea0e70e8ec2dbf158d79fc9a03c80c0f72700fc/eslint.js#L94-L97

Reply:
Updated.